### DISCUSSION TITLE
请问Growl 信息通知如何设置消息停留的时间？

### DISCUSSION THREAD

**Discussion by ApaulMyLittleAirport** [HandyOrg/HandyControl#1062] [Q&A] [answered]
请问Growl 信息通知如何设置消息停留的时间？感觉Growl.Success消息停留时间有点长，能否设置下停留时间？

**Comment by Yopler** [comment, marked as answer]
Hi 🖐,

You can try this (not sure it's working). 

```cs
GrowlInfo growlInfo = new GrowlInfo() { WaitTime = 33 };
Growl.Success(growlInfo);
```

`33 ` is just a random value

Here you can findthe source code for Growl : https://github.com/HandyOrg/HandyControl/blob/master/src/Shared/HandyControl_Shared/Controls/Growl/

**Reply by ApaulMyLittleAirport** [reply]
wo!!!!You succeeded in solving my problem,I truly appreciate your help in resolving the problem!

**Reply by CodingOctocat** [reply]
It would be nice if `Growl.xxx` supported `waitTime` overloading.

**Comment by liping94** [comment]
Why doesn't the pop-up card display an icon after using the GrowlInfo notification
